PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -5,10 +5,12 @@
 //  Created by Joshua Liebowitz on 6/18/21.
 //
 
+#import <Purchases/RCEntitlementInfo.h>
+#import <Purchases/RCPurchaserInfo.h>
+#import <Purchases/RCPurchases.h>
 #import "RevenueCatAPI.h"
-#import "Purchases.h"
-#import "RCPurchases.h"
 
+@import PurchasesCoreSwift;
```

Comment:
this import is duplicated, already present two lines below

Reply:
Oops